projects/observability/src/shared/components/span-detail/call/headers/span-detail-call-headers.component.ts

```diff
@@ -15,15 +13,23 @@ import { ScopeQueryParam } from '../../../../../pages/explorer/explorer.componen
       <ht-label [label]="this.title" class="title"></ht-label>
       <div class="container">
         <ng-container *htLoadAsync="this.records$ as records">
-          <ht-list-view [records]="records" [metadata]="metadata" display="${ListViewDisplay.Plain}" data-sensitive-pii>
+          <ht-list-view
+            [records]="records"
+            [metadata]="this.attributesMetadata"
+            display="${ListViewDisplay.Plain}"
+            data-sensitive-pii
+          >
             <div class="record-value" *htListViewValueRenderer="let record">
               <div class="value">{{ record.value }}</div>
-              <ht-explore-filter-link
-                class="filter-link"
-                [paramsOrUrl]="this.buildExploreNavigationParams | htMemoize: record | async"
+              <ht-filter-button
+                *ngIf="this.attribute && this.metadata"
+                class="filter-button"
+                [attribute]="this.attribute"
+                [metadata]="this.metadata"
```

Comment:
Consider whether we really need to have every caller of this pass metadata in. Should it (or the filter url service) just query it internally. Seems cleaner.

Reply:
done